profiles/arch/alpha/package.use.mask

```diff
@@ -502,3 +502,7 @@ net-dialup/ppp atm
 
 # Needs nut and qstat respectively, neither is keyworded on alpha.
 net-analyzer/nagios-plugins nagios-game
+
+# Vadim Misbakh-Soloviov <mva@gentoo.org> (2021-12-09)
```

Comment:
Please put new [entries](https://devmanual.gentoo.org/profiles/package.mask/index.html) at the _top_ of the mask files (below any headers).